### PR TITLE
Add contacts and about pages w/ routes

### PR DESCRIPTION
**Pull request by Francesco-Munafo**
Requested tasks completed

### CODE REVIEW

File: resources/views/about.blade.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+@extends('layouts.app')
+
+@section('main-content')
+    <div class="container">
+        <div class="row">
+            <div class="col-6 text-center">
+                <h1>About</h1>
```
Comment (fabiopacifici) [inline resources/views/about.blade.php]: Ok but write About us instead of just `About`

File: resources/views/contacts.blade.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@extends('layouts.app')
+
+@section('main-content')
+    <div class="container">
+        <div class="row">
+            <div class="col-6 text-center">
+                <h1>Contacts</h1>
```
Comment (fabiopacifici) [inline resources/views/contacts.blade.php]: Change text to Cotnact Us

File: routes/web.php [hunk 3]
```diff
@@ -16,3 +16,9 @@
 Route::get('/', function () {
```
Comment (fabiopacifici) [inline routes/web.php]: Replace the closure with a proper PageController

File: routes/web.php [hunk 4]
```diff
@@ -16,3 +16,9 @@
...
     return view('welcome');
 });
+Route::get('/about', function () {
```
Comment (fabiopacifici) [inline routes/web.php]: Replace the closure with a proper PageController

File: routes/web.php [hunk 5]
```diff
@@ -16,3 +16,9 @@
...
+    return view('about');
+});
+Route::get('/contacts', function () {
```
Comment (fabiopacifici) [inline routes/web.php]: Replace the closure with a proper PageController

**Review by fabiopacifici — CHANGES_REQUESTED** [review]
Check all comments and fix the required tasks and add the missing page controlelr

File: resources/views/about.blade.php (hunk 1 above)
Comment (Francesco-Munafo) [reply resources/views/about.blade.php]: Ok, working on it!

File: resources/views/contacts.blade.php (hunk 2 above)
Comment (Francesco-Munafo) [reply resources/views/contacts.blade.php]: Ok, working on it!

File: routes/web.php (hunk 3 above)
Comment (Francesco-Munafo) [reply routes/web.php]: Ok, working on it!

File: routes/web.php (hunk 4 above)
Comment (Francesco-Munafo) [reply routes/web.php]: Ok, working on it!

File: routes/web.php (hunk 5 above)
Comment (Francesco-Munafo) [reply routes/web.php]: Ok, working on it!

**Review by fabiopacifici — APPROVED** [review]
Thanks for your work! you are great 👍

**Pull request merged**